mlrun/track/tracker_manager.py

```diff
@@ -26,46 +27,33 @@ def __init__(self):
     def add_tracker(self, tracker: Tracker):
         """
         adds a Tracker to tracking list
+        :param tracker: class that inherits from tracker to add an instance of to trackers list
```

Comment:
No need to mention it as you have a type hint of `Tracker`.

```suggestion
        :param tracker: The tracker class to add.
```